lib/ops_preflight/server.rb

```diff
@@ -27,14 +27,14 @@ def download
     def deploy(stack_name, app_name)
       require 'ops_preflight/ops_works/deploy.rb'
 
-      OpsWorks::Deploy.new(stack_name, app_name).call(options[:release])
+      OpsWorks::Deploy.new('us-east-1', stack_name, app_name).call(options[:release])
```

Comment:
The gem already looks for `ENV['AWS_REGION']`.  I can't use it here because lots of preflight stuff runs before the env vars are available (one code path actually creates that file).